code/ashley/django/blog_proj/blog_app/templates/blog_app/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <h1>{{ message }}</h1>
+    <form action="{% url 'blog_post:BlogPost' %}" method="post">
```

Comment:
This namespace does not exist. Blog app immediately errors when attempting to load index.

> NoReverseMatch at /
> 'blog_post' is not a registered namespace